### PR TITLE
documentation/1338 - Add FPH to aircraft commands documentation

### LINKED ISSUE #1338: Add FPH to aircraft commands documentation
Apparently `fph` (fly present heading) is not in the command reference. We should add it!

### PR DESCRIPTION
**Pull request by JPmAn24**
Resolves #1338 

The purpose of this pull request is to Add FPH to aircraft commands documentation

### CODE REVIEW

File: documentation/aircraft-commands.md [hunk 1]
```diff
@@ -276,6 +277,13 @@ Altitude also accepts an `expedite` or `x` argument which can be used as the las
 
 _Syntax -_ `AAL123 c [alt]` or `AAL123 c [alt] x`
 
+## Fly Present Heading
```
Comment (Fechulo) [inline documentation/aircraft-commands.md]: This should be a h3 heading

File: documentation/aircraft-commands.md [hunk 2]
```diff
@@ -276,6 +277,13 @@ Altitude also accepts an `expedite` or `x` argument which can be used as the las
...
+_Aliases -_ `fph`
+
+_Information -_ This command has the aircraft fly at the present heading, disregarding the passing of any waypoints.
```
Comment (Fechulo) [inline documentation/aircraft-commands.md]: The phrasing is a bit strange, could you make it clearer?

Comment (erikquinn) [reply documentation/aircraft-commands.md]: Maybe to the effect of "fly straight ahead, regardless of flight plan routing" or something?

File: documentation/aircraft-commands.md (hunk 1 above)
Comment (n8rzz) [reply documentation/aircraft-commands.md]: Should also be an empty line after headings

Comment (erikquinn) [reply documentation/aircraft-commands.md]: done

File: documentation/aircraft-commands.md (hunk 2 above)
Comment (erikquinn) [reply documentation/aircraft-commands.md]: done

**Pull request merged**